### ISSUE TITLE
Update Defender Bypasses

### ISSUE THREAD

**Issue by suckma420** [UnamSanctam/UnamBinder#13]
Hey, I have seen the new miner and it bypasses perfectly for Defender so can you update it for your miner and binder as well? Thanks!

**Comment by suckma420** [comment]
@UnamSanctam

**Comment by suckma420** [comment]
@UnamSanctam Will you fix this

**Comment by suckma420** [comment]
Hello?

**Comment by UnamSanctam** [comment]
Yes sorry I slept for a bit since I was up all night, I will update both the binder and the downloader now that I've "finished" the miner.

**Comment by suckma420** [comment]
Oh, sorry for that, np, I also though that you was sleeping too because you reply really fast

**Comment by suckma420** [comment]
Any progress?

**Comment by suckma420** [comment]
How it's going

**Comment by UnamSanctam** [comment]
I'm also updating other things in the binder.

**Comment by suckma420** [comment]
OK, thanks for letting me know!
